src/app.ts

```diff
@@ -71,6 +72,14 @@ async function app() {
       false
     )
     .option('-s, --snapshot', 'generate snapshot in Structure Definition output', false)
+    .option(
+      '-c, --config <config>',
+      "override elements in sushi-config.yaml (supported: 'version', 'status', 'releaselabel') (eg: --config 'status:draft')",
```

Comment:
I just noticed that you wrap the option value in single quotes (e.g. `--config 'status:draft'`). Those are not required, as `--config status:draft` works too. Did you do it that way because values with a space in them do require that?

I don't have a strong opinion, but I don't think that FHIR allows version, status, or releaselabel values to have spaces anyway (or at least as a best practice), so it may not be necessary to show the quotes.

Reply:
Fixed